src/bot.js

```diff
@@ -1,34 +1,33 @@
 require('dotenv').config()
 const config = require('./config.json')
 const logger = require('./utils/logging.js')
-const { CommandoClient } = require('discord.js-commando');
-const path = require('path');
+const { CommandoClient } = require('discord.js-commando')
+const path = require('path')
 
 const client = new CommandoClient({
   commandPrefix: config.prefix,
   owner: process.env.BOT_OWNER,
   disableEveryone: true,
-  unknownCommandResponse: false
+  unknownCommandResponse: false,
 })
 client.registry
   .registerDefaultTypes()
   .registerGroups([
-    ["music", "Music related commands"],
-    ["code", "Code related commands"],
-    ["assistants", "Commands to assist the bot overlord(s)"],
-    ["humor", "Bot's humor processing unit"],
-    ["scrapers", "Scrape for data from remote sources"],
-    ["fetch", "Fetch data from websites"]
+    ['music', 'Music related commands'],
+    ['code', 'Code related commands'],
+    ['assistants', 'Commands to assist the bot overlord(s)'],
+    ['humor', "Bot's humor processing unit"],
+    ['scrapers', 'Scrape for data from remote sources'],
+    ['fetch', 'Fetch data from websites'],
+    ['fun', 'Games our bot likes to play'],
```

Comment:
It would be better to just stage this line and leave the style changes (single quote from double quotes) to separate commit so as to not hide them in an unrelated commit.